ext4/tar2ext4/vhdfooter.go

```diff
@@ -16,7 +20,7 @@ const (
 	diskTypeFixed          = 2
 )
 
-type vhdFooter struct {
+type VHDFooter struct {
```

Comment:
This is a breaking change in an exported package.

Reply:
hmm okay, I did this mostly for debugability with the "read-gpt" command, so I suppose we could get rid of it.

Reply:
I realized this isn't actually a breaking change, since we're going from unexported to exported. I must have misread it at first. I think you decided it's not needed though?

Reply:
Not unless there's a specific reason we want the tool to be able to read the vhd footer out.